prow/plugins/projectmanager/projectmanager_test.go

```diff
@@ -0,0 +1,284 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package projectmanager
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+type fakeGitHubClient struct {
```

Comment:
Can we use [the existing fakegithub methods](https://github.com/kubernetes/test-infra/blob/master/prow/github/fakegithub/fakegithub.go) instead of creating new ones here?

Reply:
Done